package.json

```diff
@@ -27,7 +27,8 @@
     "build": "npm run clean && npm run build-server",
     "test": "npm run eslint && npm run mocha",
     "eslint": "eslint src/. test/. --config .eslintrc.json",
-    "testrpc": "[ ! -d data/testrpc ] && mkdir -p data/testrpc; node node_modules/ethereumjs-testrpc/build/cli.node.js -d --ws --gasLimit 6500000 --db data/testrpc",
+    "testrpc-win": "mkdirp data/testrpc && node node_modules/ethereumjs-testrpc/build/cli.node.js -d --ws --gasLimit 6500000 --db data/testrpc",
```

Comment:
Can you remove the `testrpc-win` and just use `mkdirp` in the `testrpc` script. `mkdirp` seems to work with osx & I assume linux.

Reply:
Sure thing, I will update it shortly.